run_acceptance_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+ENV=${1:-local}
+BROWSER=${2:-chrome}
+DRIVER=
+
+if [ "$BROWSER" = "chrome" ]; then
+    DRIVER="-Dwebdriver.chrome.driver=/usr/local/bin/chromedriver"
+elif [ "$BROWSER" = "firefox" ]; then
+    DRIVER="-Dwebdriver.gecko.driver=/usr/local/bin/geckodriver"
+fi
+
+sbt -Dbrowser=$BROWSER -Denvironment=$ENV $DRIVER acceptance:test
```

Comment:
I don't know if it would blow up if both weren't present, but wonder if we could refactor to remove the conditional and allow overwriting the path to the webdrivers like this (untested)

```
#!/bin/bash -e
ENV=${1:-local}
BROWSER=${2:-chrome}
CHROME_WEB_DRIVER=${CHROME_WEB_DRIVER:-/usr/local/bin/chromedriver}
GECKO_WEB_DRIVER=${GECKO_WEB_DRIVER:-/usr/local/bin/geckodriver}

sbt -Dbrowser=$BROWSER \
    -Denvironment=$ENV \
    -Dwebdriver.chrome.driver=$CHROME_WEB_DRIVER \
    -Dwebdriver.gecko.driver=$GECKO_WEB_DRIVER \
    acceptance:test
```